docs/components/Intro.js

```diff
@@ -350,317 +199,220 @@ const displayLinks = (links) => {
-            fill="currentColor"
-          />
-        </svg>
-        <h2 className="mb-4 text-xl font-semibold">SDK how-to guides (Legacy)</h2>
-      </div>
-      <p className="mb-4 flex font-medium">SDK specific implementation guides and examples.</p>
-      <div className="grid grid-cols-2">
+    <div className="main-card">
+      <h1>{title}</h1>
+      <p>{content}</p>
+      <ul className="single-column-list">{displayLinks(links)}</ul>
+    </div>
+  )
+}
+
+
+function CloudCard({ links }) {
+  return (
+    <div className="main-card">
+      <div className="grid grid-cols-1 md:grid-cols-2">
         <div>
-          <ul className="flex flex-col space-y-2">
-            {/* <ul className="grid grid-cols-2 gap-6 xl:gap-8"> */}
-            <li className="">
-              <Link className="flex items-center space-x-3 hover:underline" to={useBaseUrl("/go/")}>
-                <img
-                  className="h-8 w-8 transition hover:scale-110 code-logo"
-                  src="/img/golang.svg"
-                  alt="Go lang logo"
-                />
-                <p className="font-normal">Go</p>
-              </Link>
-            </li>
-            <li className="">
-              <Link className="flex items-center space-x-3 hover:underline" to={useBaseUrl("/java")}>
-                <img className="h-8 w-8 transition hover:scale-110 code-logo" src="/img/java.svg" alt="Java logo" />
-                <p className="font-normal">Java</p>
-              </Link>
-            </li>
-          </ul>
+          <h1>Temporal Cloud</h1>
+          <p>Temporal Cloud tools and how-to guides.</p>
+          <ul className="landing-card-list">{displayLinks(links)}</ul>
         </div>
-
         <div>
-          <ul className="flex flex-col space-y-2">
-            {/* <ul className="grid grid-cols-2 gap-6 xl:gap-8"> */}
-            <li className="">
-              <Link className="flex items-center space-x-5" to={useBaseUrl("/dev-guide/php")}>
-                <div className="flex items-center space-x-3 hover:underline">
-                  <img className="h-8 w-8 transition hover:scale-110 code-logo" src="/img/php.svg" alt="PHP logo" />
-                  <p className="font-normal">PHP</p>{" "}
-                </div>
-              </Link>
-            </li>
-            <li className="">
-              <Link to={useBaseUrl("/typescript/introduction")}>
-                <div className="flex items-center space-x-3 hover:underline">
-                  <img
-                    className="h-7 w-7 pl-1 transition hover:scale-110 code-logo"
-                    src="/img/typescript.svg"
-                    alt="TypeScript logo"
-                  />
-                  <p className="font-normal">TypeScript</p>
-                </div>
-              </Link>
-            </li>
-          </ul>
+          <h1>Get started with Cloud</h1>
```

Comment:
```suggestion
          <h1>Is Temporal Cloud right for you?</h1>
```

I think we need to update this header to match the call to action.